PokemonGoMap/PokemonGo-Map
# Manual Captcha Solving

## Description
Manual Captcha Solving without 2captcha

## Motivation and Context
Someone may not have 2captcha, they need to solve the captcha themselves, it provides a easy way for them to solve the captcha

## How Has This Been Tested?




1. Replace __domain__ with your PokemonGoMap domain and add a bookmarklet, provide your PokemonGoMap domain to runserver.py using -mcsd or --manual-captcha-solving-domain

javascript:(function (){document.getElementsByTagName('head')[0].appendChild(document.createElement('script')).src='__domain__/inject.js?'+Math.random();}());

2. Wait for an account to encounter the captcha

3. Go to https://pgorelease.nianticlabs.com/

4. Use the bookmarklet above

<img width="170" alt="screen shot 2016-11-16 at 9 50 54 pm" src="https://cloud.githubusercontent.com/assets/918554/20349762/ceac0414-ac46-11e6-9576-cab57a4968e0.png">

5. Solve the recaptcha

6. Recaptcha token will be saved to Database in Token table

7. token_request will get a token which created after the request made and return to the search worker and verify the captcha challenge

p.s.: if token_request cannot get a token which created after the request made within manual-captcha-solving-allowance-time(default: 60s), token_request will return 'ERROR' and append the account to account_failures

## Screenshots (if appropriate):

## Types of changes


## Checklist:
